src/probnum/filtsmooth/gaussfiltsmooth/kalmanposterior.py

```diff
@@ -45,30 +47,31 @@ def state_rvs(self):
         """
         return self._state_rvs
 
-    def __call__(self, t, smoothed=True):
+    def __call__(self, t):
         """
         Evaluate the time-continuous posterior at location `t`
 
         Algorithm:
         1. Find closest t_prev and t_next, with t_prev < t < t_next
         2. Predict from t_prev to t
-        3. (if `smoothed=True`) Predict from t to t_next
-        4. (if `smoothed=True`) Smooth from t_next to t
+        3. (if `self._with_smoothing=True`) Predict from t to t_next
+        4. (if `self._with_smoothing=True`) Smooth from t_next to t
         5. Return random variable for time t
 
         Parameters
         ----------
         t : float
             Location, or time, at which to evaluate the posterior.
-        smoothed : bool, optional
-            If ``True`` (default) perform smooth interpolation. If ``False`` perform a
-            prediction from the previous location, without smoothing.
 
         Returns
         -------
         :obj:`RandomVariable`
             Estimate of the states at time ``t``.
         """
+        if np.isscalar(t) is False:
```

Comment:
"pythonic" would be: `if not np.isscalar(t)` ;) I'm surprised pylint didn't compain (yet?)

Reply:
True.

I guess we excluded so many pylint warnings globally that it doesnt complain about anything anymore... :D